packages/json-schema/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@loopback/json-schema",
+  "version": "4.0.0-alpha.1",
+  "description": "Converts TS classes into JSON Schemas using TypeScript's reflection API",
+  "engines": {
+    "node": ">=8"
+  },
+  "scripts": {
+    "build": "npm run build:dist && npm run build:dist6",
+    "build:current": "lb-tsc",
+    "build:dist": "lb-tsc es2017",
+    "build:dist6": "lb-tsc es2015",
+    "build:apidocs": "lb-apidocs",
+    "clean": "lb-clean loopback-json-schema*.tgz dist dist6 package api-docs",
+    "prepare": "npm run build && npm run build:apidocs",
+    "pretest": "npm run build:current",
+    "test": "lb-dist mocha --opts ../../test/mocha.opts 'DIST/test/unit/**/*.js' 'DIST/test/integration/**/*.js' 'DIST/test/acceptance/**/*.js'",
```

Comment:
If we are supporting Node.js `>=8`, then there is no need for `lb-dist`, because there is only a single `dist` folder.